### ISSUE TITLE
Regression affecting standard library: Data/List/Relation/Unary/Any/Properties: B.b != q₁ of type Level

### ISSUE THREAD

**Issue by andreasabel** [agda/agda#4351]
This error is reported for the standard library after either `make std-lib` or `make up-to-date-std-lib` or `make fast-forward-std-lib`:
```
Data/List/Relation/Unary/Any/Properties.agda:251,16-27
B.b != q₁ of type Level
when checking that the inferred type of an application
  Any.map _f_1489 y∈ys ≡ pq′
matches the expected type
  Any.map (λ q₂ → p , q₂) (Any.map (λ x₁ → P.subst Q x₁ _1452) y∈ys)
  ≡ pq′
```
This is probably a duplicate issue report; I have seen this regression mentioned before but can't find it now.

What is the correct incantation to get a version of the standard library that works with the current master of Agda?

**Comment by gallais** [comment]
`git checkout experimental` in the stdlib directory should give you what you
are looking for.

**Comment by andreasabel** [comment]
> `git checkout experimental` in the stdlib directory should give you what you
> are looking for.

Thanks.  I think `up-to-date-std-lib` used to give the correct version and should maybe restored to this purpose...

**Comment by andreasabel** [comment]
Even with experimental,
commit 6a5176460f7cba106cf3b40420bba4ea1682e4f4 (HEAD -> experimental, origin/experimental)
Author: Ulf Norell 
Date:   Fri Dec 20 09:29:38 2019 +0100

Add implicit arguments required after fix of agda/agda#4174

I get the error here:
https://github.com/agda/agda-stdlib/blob/3c12349066593f41e48afe6dde61b8a34fda38f5/src/Data/List/Relation/Unary/Any/Properties.agda#L251
Strangely enough, `make library-test` runs fine, but within emacs or from my project it fails.

**Comment by andreasabel** [comment]
Ah, actually Ulf's commit fixes this problem (knew it!), but I had some confusion here.
